test/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.FunctionalTests/Infrastructure/ApplicationTestFixture.cs

```diff
@@ -30,24 +31,18 @@ protected ApplicationTestFixture(string applicationName)
 
         public ILogger Logger { get; private set; }
 
-        public DeploymentResult CreateDeployment()
-        {
-            lock (_deploymentLock)
```

Comment:
Curious why was this required earlier? All the tests publish to different output folders, so even if they run in parallel it shouldn't be a problem